content/en/guides/configuration-glossary/configuration-components.md

```diff
@@ -8,13 +8,253 @@ position: 5
 
 > Nuxt.js 2.13+ can scan and auto import your components.
 
+<base-alert>It is possible to use this feature with Nuxt 2.10 - 2.12. Just manually install and add `@nuxt/components` to `buildModules` inside `nuxt.config`.</base-alert>
```

Comment:
```suggestion
<base-alert>It is possible to use this feature with Nuxt 2.10 - 2.12 if you cannot upgrade. Just manually install and add `@nuxt/components` to `buildModules` inside `nuxt.config`.</base-alert>
```

Reply:
(or we can just not mention this to be consistent with features section that is saying 2.13+ needed)